apps/web/app/routes/get-started.tsx

```diff
@@ -247,17 +247,26 @@ export default function Component() {
       <p>
         Domain Functions will parse the request&apos;s <em>formData</em> and
         perform the mutation only if everything is valid. If something goes bad,
-        it will return structured error messages for us.
+        it will return structured error messages for us. You can also put this
+        in the route where you'll be submitting a form.
```

Comment:
Actually this is not a good practice as domain functions usually have server side code.
I use to put my schemas in a `app/domain/some-domain` folder so it can be used in the domain-function and I also keep my domain functions away from the route as there's no need (and often there are problems) to send that code to the client bundle.

Now that we have route folders in Remix you could add those functions in separate files under that route's folder but it is important to make sure you only use them in either loaders or actions so it won't be sent to the client.

Reply:
Good point. I'm writing short mutations inline in the `formAction`, but worth clarifying! Let me update the patch.